src/web/core/src/main/java/org/geoserver/web/data/store/ParamInfo.java

```diff
@@ -105,7 +108,11 @@ public String getTitle() {
     public boolean isPassword() {
         return password;
     }
-    
+
+    public String getLevel() {
```

Comment:
thanks, working around ParamInfo is frustrating, do we need javadocs for these public methods (say to define the levels).